docs/getting-started-guides/docker-multinode/master.sh

```diff
@@ -75,6 +75,7 @@ detect_lsb() {
     fi
 
     lsb_dist="$(echo ${lsb_dist} | tr '[:upper:]' '[:lower:]')"
+    echo "Detected $lsb_dist"
```

Comment:
If that's the case, it definitely is. 
Yet currently we only support limited platforms (debian, ubuntu, centos, amzn). If this works for them all, i believe it's fine for now. (should be well doc though)
The original idea of this lsb_release detection cames from docker installation guide, FYI.